### PR TITLE
Country Info webapp final 

### PR DESCRIPTION
**Pull request by iankiema**
The following files were added to this project
_Homepage file
-Details file
-Testing files for the homepage, details page, slice, and search file.
-Linter files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,160 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+<br>
+
+- [View Video](https://www.loom.com/share/My-Project-Presentation-41a30e8e8813440bb67feb7c13a5c314?sid=1eadfa42-6ddd-4444-a8b1-060bdc9da551)
+
+- [View Live Demo](https://github.com/iankiema/Country-Info-webapp.git)
```
Comment (TedLivist) [inline README.md]: - [ ] Kindly deploy your app and provide its link.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @iankiema,

Good job so far!
✔️ Implementation of search functionality
✔️ Implementation unique routes for each country
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,160 @@
...
+
+<br>
+
+- **React-Based SPA**
+- **Up-to-Date Country stats**
+- **Effortless Country Search**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
```
Comment (LionRouge1) [inline README.md:51]: - [ ] As mentioned in the previous review, It requested in the project requirement that the project should be deployed and accessible online using [Render](https://render.com/), [Netlify](https://www.netlify.com/) or [GitHub Pages](https://pages.github.com/). And we should add the link to the online version of the app to the README file.
Screenshot from 2023-09-14 11-25-16

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @iankiema, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
almost

## Highlights
- Recorded a video presenting the features of the project :heavy_check_mark: 
- The project contains a `Home` page and a `Detail` page :heavy_check_mark: 
- App has unit tests for several components of the app and some pure functions :heavy_check_mark: 

## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (Whoistolu) [reply README.md:51]: - Project deployment complete

**Review by Whoistolu — APPROVED** [review]
### Hi @iankiema 

Great job deploying the project as recommended by the previous reviewers 👍🏽 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**